stable/postgresql/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if and .Values.metrics.enabled .Values.metrics.serviceMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "postgresql.fullname" . }}
+  labels:
+    app: {{ template "postgresql.name" . }}
+    chart: {{ template "postgresql.chart" . }}
+    release: {{ .Release.Name | quote }}
+    heritage: {{ .Release.Service | quote }}
+    {{- if .Values.metrics.serviceMonitor.additionalLabels }}
+{{ toYaml .Values.metrics.serviceMonitor.additionalLabels | indent 4 }}
+    {{- end }}
+spec:
+  endpoints:
+    - port: metrics
+      interval: 30s
```

Comment:
This should be configurable. See https://github.com/helm/charts/blob/master/stable/mariadb/templates/servicemonitor.yaml#L24

Reply:
*scrapeTimeout* should be configurable. See https://github.com/helm/charts/blob/master/stable/mariadb/templates/servicemonitor.yaml#L27

Reply:
With default values or with default Prometheus values, as in these PR #14279

Reply:
With default Prometheus values, as done at that PR

Reply:
Done